src/index.js

```diff
@@ -183,14 +191,16 @@ function fakeSignature (prefix, flowType, opts) {
   }
 }
 
-function flowType (prefix, prop, opts) {
-  return getFakeFlow(prefix, prop.flowType, opts)
+function typed (prefix, prop, opts) {
+  return getFakeTyped(prefix, prop.flowType || prop.tsType, opts)
 }
 
-function getFakeFlow (prefix, flowType, opts) {
-  switch (flowType.name) {
+function getFakeTyped (prefix, type, opts) {
+  switch (type.name) {
     case 'boolean':
       return fakeBool()
+    case 'tuple':
```

Comment:
nothing really major happening on this file, I only renamed a bunch of funcions, and added this `tuple` case here to support props like this: 
```
type Props = {
  test: []
}
```

empty arrays as props are named as `tuple`, and were falling into the `unknown` case.